### ISSUE TITLE
New submissions for Fri, 29 Sep 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#114]
# New submissions for Fri, 29 Sep 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### Simulating ionization feedback from young massive stars: impact of  numerical resolution
 - **Authors:** Yunwei Deng, Hui Li, Rahul Kannan, Aaron Smith, Mark Vogelsberger, Greg L. Bryan
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2309.15900
 - **Pdf link:** https://arxiv.org/pdf/2309.15900
 - **Abstract**
 Modelling galaxy formation in hydrodynamic simulations has increasingly adopted various radiative transfer methods to account for photoionization feedback from young massive stars. However, the evolution of HII regions around stars begins in dense star-forming clouds and spans large dynamical ranges in both space and time, posing severe challenges for numerical simulations in terms of both spatial and temporal resolution that depends strongly on gas density ($\propto n^{-1}$). In this work, we perform a series of idealized HII region simulations using the moving-mesh radiation-hydrodynamic code Arepo-RT to study the effects of numerical resolution. The simulated results match the analytical solutions and the ionization feedback converges only if the Str\"omgren sphere is resolved by at least $10$--$100$ resolution elements and the size of each time integration step is smaller than $0.1$ times the recombination timescale. Insufficient spatial resolution leads to reduced ionization fraction but enhanced ionized gas mass and momentum feedback from the HII regions, as well as degrading the multi-phase interstellar medium into a diffuse, partially ionized, warm ($\sim8000$\,K) gas. On the other hand, insufficient temporal resolution strongly suppresses the effects of ionizing feedback. This is because longer timesteps are not able to resolve the rapid variation of the thermochemistry properties of the gas cells around massive stars, especially when the photon injection and thermochemistry are performed with different cadences. Finally, we provide novel numerical implementations to overcome the above issues when strict resolution requirements are not achievable in practice.
#### MESA-Web: A cloud resource for stellar evolution in astronomy curricula
 - **Authors:** Carl E. Fields, Richard H. D. Townsend, A. L. Dotter, Michael Zingale, F. X. Timmes
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Astrophysics of Galaxies (astro-ph.GA); High Energy Astrophysical Phenomena (astro-ph.HE); Instrumentation and Methods for Astrophysics (astro-ph.IM); Physics Education (physics.ed-ph)
 - **Arxiv link:** https://arxiv.org/abs/2309.15930
 - **Pdf link:** https://arxiv.org/pdf/2309.15930
 - **Abstract**
 We present MESA-Web, a cloud resource with an online interface to the Modules for Experiments in Stellar Astrophysics (MESA) software instrument. MESA-Web allows learners to evolve stellar models without the need to download and install MESA. Since being released in 2015, MESA-Web has delivered over 17,000 calculations to over 2,200 unique learners and currently performs about 11 jobs per day. MESA-Web can be used as an educational tool for stars in the classroom or for scientific investigations. We report on new capabilities of MESA-Web introduced since its 2015 release including learner-supplied nuclear reaction rates, custom stopping conditions, and an expanded selection of input parameters. To foster collaboration we have created a Zenodo MESA-Web community hub where instructors can openly share examples of using MESA-Web in the classroom. We discuss two examples in the current community hub. The first example is a lesson module on Red Giant Branch stars that includes a suite of exercises designed to fit a range of learners and a Jupyter workbook for additional analysis. The second example is lesson materials for an upper-level Astronomy majors course in Stars and Radiation that includes an assignment verifying some of the expected trends that are presented in a popular stellar physics textbook.
#### Tides in clouds: control of star formation by long-range gravitational  force
 - **Authors:** Guang-Xing Li
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2309.16125
 - **Pdf link:** https://arxiv.org/pdf/2309.16125
 - **Abstract**
 Gravity drives the collapse of molecular clouds through which stars form, yet the exact role of gravity in cloud collapse remains a complex issue. Studies point to a picture where star formation occurs in clusters. In a typical, pc-sized cluster-forming region, the collapse is hierarchical, and the stars should be born from regions of even smaller sizes ($\approx 0.1\;\rm pc$). The origin of this spatial arrangement remains under investigation. Based on a high-quality surface density map towards the Perseus region, we construct a 3D density structure, compute the gravitational potential, and derive eigenvalues of the tidal tensor ($\lambda_{\rm min}$, $\lambda_{\rm mid}$, $\lambda_{\rm max}$, $\lambda_{\rm min} < \lambda_{\rm mid} < \lambda_{\rm max}$), analyze the behavior of gravity at every location and reveal its multiple roles in cloud evolution. We find that fragmentation is limited to several isolated, high-density ``islands''. Surrounding them, is a vast amount of gas ($75 \%$ of the mass, $95 \%$ of the volume) that stays under the influence of extensive tides where fragmentation is suppressed. This gas will be transported towards these regions to fuel star formation. The spatial arrangement of regions under different tides explains the hierarchical and localized pattern of star formation inferred from the observations. Tides were first recognized by Newton, yet this is the first time its dominance in cloud evolution has been revealed. We expect this link between cloud density structure and role gravity to be strengthened by future studies, resulting in a clear view of the star formation process.
#### Astroconformer: The Prospects of Analyzing Stellar Light Curves with  Transformer-Based Deep Learning Models
 - **Authors:** Jia-Shu Pan, Yuan-Sen Ting, Jie Yu
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Earth and Planetary Astrophysics (astro-ph.EP); Instrumentation and Methods for Astrophysics (astro-ph.IM); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2309.16316
 - **Pdf link:** https://arxiv.org/pdf/2309.16316
 - **Abstract**
 Light curves of stars encapsulate a wealth of information about stellar oscillations and granulation, thereby offering key insights into the internal structure and evolutionary state of stars. Conventional asteroseismic techniques have been largely confined to power spectral analysis, neglecting the valuable phase information contained within light curves. While recent machine learning applications in asteroseismology utilizing Convolutional Neural Networks (CNNs) have successfully inferred stellar attributes from light curves, they are often limited by the local feature extraction inherent in convolutional operations. To circumvent these constraints, we present $\textit{Astroconformer}$, a Transformer-based deep learning framework designed to capture long-range dependencies in stellar light curves. Our empirical analysis, which focuses on estimating surface gravity ($\log g$), is grounded in a carefully curated dataset derived from $\textit{Kepler}$ light curves. These light curves feature asteroseismic $\log g$ values spanning from 0.2 to 4.4. Our results underscore that, in the regime where the training data is abundant, $\textit{Astroconformer}$ attains a root-mean-square-error (RMSE) of 0.017 dex around $\log g \approx 3 $. Even in regions where training data are sparse, the RMSE can reach 0.1 dex. It outperforms not only the K-nearest neighbor-based model ($\textit{The SWAN}$) but also state-of-the-art CNNs. Ablation studies confirm that the efficacy of the models in this particular task is strongly influenced by the size of their receptive fields, with larger receptive fields correlating with enhanced performance. Moreover, we find that the attention mechanisms within $\textit{Astroconformer}$ are well-aligned with the inherent characteristics of stellar oscillations and granulation present in the light curves.
#### Evidence of a Cloud-Cloud Collision from Overshooting Gas in the  Galactic Center
 - **Authors:** Savannah R. Gramze, Adam Ginsburg, David S. Meier, Juergen Ott, Yancy Shirley, Mattia C. Sormani, Brian E. Svoboda
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2309.16403
 - **Pdf link:** https://arxiv.org/pdf/2309.16403
 - **Abstract**
 The Milky Way is a barred spiral galaxy with "bar lanes" that bring gas towards the Galactic Center. Gas flowing along these bar lanes often overshoots, and instead of accreting onto the Central Molecular Zone, it collides with the bar lane on the opposite side of the Galaxy. We observed G5, a cloud which we believe is the site of one such collision, near the Galactic Center at (l,b) = (+5.4, -0.4) with the ALMA/ACA. We took measurements of the spectral lines $^{12}$CO J=2-1, $^{13}$CO J=2-1, C$^{18}$O J=2-1, H$_2$CO J=3$_{03}$-2$_{02}$, H$_{2}$CO J=3$_{22}$-2$_{21}$, CH$_{3}$OH J=4$_{22}$-3$_{12}$, OCS J=18-17 and SiO J=5-4. We observed a velocity bridge between two clouds at $\sim$50 km/s and $\sim$150 km/sin our position-velocity diagram, which is direct evidence of a cloud-cloud collision. We measured an average gas temperature of $\sim$60 K in G5 using H$_2$CO integrated intensity line ratios. We observed that the $^{12}$C/$^{13}$C ratio in G5 is consistent with optically thin, or at most marginally optically thick $^{12}$CO. We measured 1.5 x 10$^9$ cm$^{-2}$(K km/s)$^{-1}$ for the local X$_{CO}$, 10-20x less than the average Galactic value. G5 is strong direct observational evidence of gas overshooting the Central Molecular Zone (CMZ) and colliding with a bar lane on the opposite side of the Galactic center.
#### Improvement of digest2 NEO Classification Code -- utilizing the  Astrometry Data Exchange Standard
 - **Authors:** Peter Vereš, Richard Cloete, Robert Weryk, Abraham Loeb, Matthew J. Payne
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2309.16407
 - **Pdf link:** https://arxiv.org/pdf/2309.16407
 - **Abstract**
 We describe enhancements to the digest2 software, a short-arc orbit classifier for heliocentric orbits. Digest2 is primarily used by the Near-Earth Object (NEO) community to flag newly discovered objects for a immediate follow-up and has been a part of NEO discovery process for more than 15 years. We have updated the solar system population model used to weight the digest2 score according to the 2023 catalog of known solar system orbits and extended the list of mean uncertainties for 140 observatory codes. Moreover, we have added Astrometry Data Exchange Standard (ADES) input format support to digest2, which provides additional information for the astrometry, such as positional uncertainties for each detection. The digest2 code was also extended to read the roving observer astrometric format as well as the ability to compute a new parameter from the provided astrometric uncertainties ($RMS'$) that can serve as an indicator of in-tracklet curvature when compared with tracklet's great-circle fit RMS. Comparison with the previous version of digest2 confirmed the improvement in accuracy of NEO identification and found that using ADES XML input significantly reduces the computation time of the digest2.
#### Fragmentation and dynamics of dense gas structures in the proximity of  massive young stellar object W42-MME
 - **Authors:** N. K. Bhadari, L. K. Dewangan, L. E. Pirogov, A. G. Pazukhin, I. I. Zinchenko, A. K. Maity, Saurabh Sharma
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2309.16472
 - **Pdf link:** https://arxiv.org/pdf/2309.16472
 - **Abstract**
 We present an analysis of the dense gas structures in the immediate surroundings of the massive young stellar object (MYSO) W42-MME, using the high-resolution (0$''$.31$\times$0$''$.25) ALMA dust continuum and molecular line data. We performed a dendrogram analysis of H$^{13}$CO$^{+}$ (4-3) line data to study multi-scale structures and their spatio-kinematic properties, and analyzed the fragmentation and dynamics of dense structures down to $\sim$2000 AU scale. Our results reveal 19 dense gas structures, out of which 12 are leaves and 7 are branches in dendrogram terminology. These structures exhibit transonic-supersonic gas motions (1$<\mathcal{M}<5$) with overvirial states ($\alpha_{\rm vir}\geq2$). The non-thermal velocity dispersion-size relation ($\sigma_{\rm nt}-L$) of dendrogram structures shows a weak negative correlation, while the velocity dispersion across the sky ($\delta\mathit{V_{\rm lsr}}$) correlates positively with structure size ($L$). Velocity structure function ($S_{2}(l)^{1/2}$) analysis of H$^{13}$CO$^{+}$ data reveals strong power-law dependencies with lag ($l$) up to a scale length of $\lesssim$ 6000 AU. The mass-size ($M-R$) relation of dendrogram structures shows a positive correlation with power-law index of 1.73$\pm$0.23, and the leaf L17 hosting W42-MME meets the mass-size conditions for massive star formation. Blue asymmetry is observed in the H$^{12}$CO$^{+}$ (4-3) line profiles of most of the leaves, indicating infall. Overall, our results observationally support the hierarchical and chaotic collapse scenario in the proximity of the MYSO W42-MME.
#### Intrinsic Emission of PSR B1937+21 at 327 MHz
 - **Authors:** Nikhil Mahajan, Marten H. van Kerkwijk
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2309.16525
 - **Pdf link:** https://arxiv.org/pdf/2309.16525
 - **Abstract**
 At 327 MHz, the observed emission of PSR B1937+21 is greatly affected by scattering in the interstellar medium, on a timescale of order the pulse period. We use the bright impulsive giant pulses emitted by the pulsar to measure the impulse response of the interstellar medium and then recover the intrinsic emission of the pulsar by deconvolution -- revealing fine structure on timescales not normally observable. We find that the intrinsic widths of the main pulse and interpulse in the pulse profile are similar to those measured at higher frequencies. We detect 60,270 giant pulses which typically appear as narrow, ~100 ns bursts consisting of one to few nanoshots with widths $\lesssim \! 10$ ns. However, about 10% of the giant pulses exhibit multiple bursts which seem to be causally related to each other. We also report the first detection of giant micropulses in PSR B1937+21, primarily associated with the regular main pulse emission. These are distinct from giant pulses not only in the phases at which they occur, but also in their larger widths, of order a microsecond, and steeper energy distribution. These measurements place useful observational constraints on emission mechanisms for giant pulses as well as the regular radio emission of millisecond pulsars.
#### 1-arcsecond imaging strategy for the LoTSS survey using the  International LOFAR Telescope
 - **Authors:** Haoyang Ye, Frits Sweijen, Reinout van Weeren, Wendy Williams, Jurjen de Jong, Leah K. Morabito, Huub Rottgering, T. W. Shimwell, P.N. Best, Marco Bondi, Marcus Brüggen, Francesco de Gasperin, C. Tasse
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2309.16560
 - **Pdf link:** https://arxiv.org/pdf/2309.16560
 - **Abstract**
 We present the first wide area (2.5 x 2.5 square degrees), deep (median noise of approximately 80 microJy per beam) LOFAR High Band Antenna image at a resolution of 1.2 arcseconds by 2 arcseconds. It was generated from an 8-hour International LOFAR Telescope (ILT) observation of the ELAIS-N1 field at frequencies ranging from 120 to 168 MHz with the most up-to-date ILT imaging strategy. This intermediate resolution falls between the highest possible resolution (0.3 arcseconds) achievable by using all International LOFAR Telescope (ILT) baselines and the standard 6-arcsecond resolution in the LoTSS (LOFAR Two-meter Sky Survey) image products utilizing the LOFAR Dutch baselines only. This is the first demonstration of the feasibility of approximately 1 arcsecond imaging using the ILT, providing unique information on source morphology at scales below the surface brightness limits of higher resolutions. The total calibration and imaging time is approximately 52,000 core hours, nearly five times more than producing a 6-arcsecond image. We also present a radio source catalog containing 2263 sources detected over the 2.5 x 2.5 square degrees image of the ELAIS-N1 field, with a peak intensity threshold of 5.5 sigma. The catalog has been cross-matched with the LoTSS deep ELAIS-N1 field radio catalog, and its flux density and positional accuracy have been investigated and corrected accordingly. We find that approximately 80% of sources that we expect to be detectable based on their peak brightness in the LoTSS 6-arcsecond image are detected in this image, which is approximately a factor of two higher than for 0.3 arcsecond imaging in the Lockman Hole, implying there is a wealth of information on these intermediate scales.

by olozhika (Xing Yuchen). 

2023-09-29
